test/EmptyApp.FunctionalTests/Startup.cs

```diff
@@ -46,6 +47,14 @@ public void Configure(IApplicationBuilder app)
                         telemetryClient.TrackTrace("Fetched contact details.", SeverityLevel.Information);
                         await context.Response.WriteAsync("Hello!");
                     }
+                    else if (context.Request.GetUri().ToString().Contains("Dependency"))
+                    {
+                        using (HttpClient client = new HttpClient())
+                        {
+                            await client.GetAsync("https://www.microsoft.com/");
```

Comment:
I think microsoft.com was so widely used by companies as their ping test destination that MSIT turned off ping responses a few years ago.  I realize this is not a ping but the spirit is the same, we shouldn't abuse our network for testing if there is another option.